server/src/model/_cli.ts

```diff
@@ -121,51 +129,57 @@ async function lintLinks(bookDirs: string[]) {
             new URL(url)
           } catch {
             hasErrors = true
-            log(`Error: Could not parse URL '${url}' and urlEncoded to show any odd unicode characters`, encodeURI(url))
+            logCsv(bundle, page, link, 'parse', encodeURI(url))
             continue
           }
 
-          proto.get(url, res => {
-            if (res.statusCode !== undefined) {
-              if (res.statusCode >= 200 && res.statusCode < 300) {
-                console.log('Ok:', res.statusCode, link.url)
-              } else if (res.statusCode >= 300 && res.statusCode < 400) {
-                log('Following Redirect:', res.statusCode, link.url)
-                const destUrl = res.headers.location
-                if (destUrl !== undefined) {
+          let destUrl = ''
+          try {
+            const res = await fetch(url, { redirect: 'manual' })
```

Comment:
Out of curiosity, is there a reason to make a `GET` request instead of a `HEAD` request?

Reply:
I assumed not all websites would respond to `HEAD` requests  but it's much cheaper to make `HEAD` requests so I switched the code; thanks!